.github/workflows/build-and-deploy-prod.yml

```diff
@@ -92,10 +92,18 @@ jobs:
               id: task-def-plugins
```

Comment:
Would be helpful to update the `name` (line 91) to reflect that this is related to `plugins-ingestion` work